app/locales/en.json

```diff
@@ -24,8 +24,8 @@
   "About the service": "About the service",
   "In connection with the outbreak of COVID-19, many people have been tested for the virus, but due to lack of resources, the overwhelming majority have not been tested. It is therefore assumed that there are large missing numbers here in connection with the outbreak.": "In connection with the outbreak of COVID-19, many people have been tested for the virus, but due to lack of resources, the overwhelming majority have not been tested. It is therefore assumed that there are large missing numbers here in connection with the outbreak.",
   "Using self-report, you can get a better picture of how many people have symptoms without exposing health workers to potential contamination hazards and without using up valuable infection control equipment that is already in short supply. We at Bustbyte, well helped by other volunteers, have created this tool in response to": "Using self-report, you can get a better picture of how many people have symptoms without exposing health workers to potential contamination hazards and without using up valuable infection control equipment that is already in short supply. We at Bustbyte, well helped by other volunteers, have created this tool in response to",
-  "this article": "this article",
-  "as an attempt to chart these numbers.": "as an attempt to chart these numbers.",
+  "this article": "Si è preso spunto da questo articolo norgevese",
```

Comment:
Woops, this should probably be reverted

Reply:
it is correct: 

```
  "this article": "da questo articolo norgevese",
  "as an attempt to chart these numbers.": "e la piattaforma si è iniziato a sviluppare per raccogliere questi dati.",
```

we changed a little bit the italian translation to better fit our "italian" minds ;) 
here we say that the platform has been inspired by the norwegian article

Reply:
Yes, but this is in the english locale

Reply:
should be fixed